LowerSchool.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="Description" content="INF 7420 Project">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Document</title>
```

Comment:
I recommend changing the title of the pages.

Reply:
Dear Axa, I am hoping that my request for your review has gone through. I realize that I am behind and am doing my best. Thank you, Jen